web/gdpr.js

```diff
@@ -1,13 +1,11 @@
-import shopify from "./shopify.js";
-
-export function setupGDPRWebHooks() {
+export const GDPRWebhookHandlers = [
```

Comment:
With this setup, is it easy / possible to grab the app object to e.g. fire queries in these handlers?